addons/orbat/XEH_postInit.sqf

```diff
@@ -97,32 +97,13 @@ FUNC(PFHUpdate) = {
 
 
 // Force a 1 second wait.
-[{
-    if ([] call BIS_fnc_isLoading) exitWith {};
-    [_this select 1] call CBA_fnc_removePerFrameHandler;
-    [{
-        //Ensure a 0.5 second delay.
-        params ["_params"];
-        _params params ["_tickTime"];
-        if (diag_tickTime < _tickTime + 1.5) exitWith {};
-        
-        [_this select 1] call CBA_fnc_removePerFrameHandler; 
-        
-        [{
-            if (isNull player) exitWith {};
-            if (isNil QEGVAR(common,VarSync)) exitWith {};
-            [_this select 1] call CBA_fnc_removePerFrameHandler;
-
-            [player, true] call FUNC(setup);
-            [player] call FUNC(createBriefingPage);
-        }, 0.5] call CBA_fnc_addPerFrameHandler;  
-    },0.5,[diag_tickTime]] call CBA_fnc_addPerFrameHandler;
-}, 0.5] call CBA_fnc_addPerFrameHandler;  
-
-
-
-//Spawn thread to update fireteam positions overtime.
-
-// Do an update to set initial positions, then add PFH.
-[] call FUNC(PFHUpdate);
-[FUNC(PFHUpdate), GVAR(markerUpdateInterval), []] call CBA_fnc_addPerFrameHandler;
+[{!([] call BIS_fnc_isLoading) && {!isNull player} && {!isNil QEGVAR(common,VarSync)}},{
+    [player, true] call FUNC(setup);
+    [player] call FUNC(createBriefingPage);
+
+    //Spawn thread to update fireteam positions overtime.
+
+    // Do an update to set initial positions, then add PFH.
+    [] call FUNC(PFHUpdate);
```

Comment:
I am a bit nervous about changing the conditions here, as there's a few potential synchronisation events. As this is running during the briefing phase, I know I had my reasons at the the time for the timed delay, it might have been to allow players to be loaded in and showing up on the ORBAT.

Removing the timed delay would require some MP testing on a dedicated server.

Reply:
I've tested it on a local dedicated server through https://community.bistudio.com/wiki/Multiplayer_Scripting#Testing_locally. I'm not encountering any issues there apart from the one I fixed in the commit below though it might be different with more players and ping.